radis/lbl/factory.py

```diff
@@ -1196,6 +1494,70 @@ def non_eq_spectrum(
             self._clean_temp_file()
             raise
 
+    def _get_log_2gs(self):
+        """ Returns log_2gs if it already exists in the dataframe, otherwise computes it using gamma_air """
+        df = self.df0
```

Comment:
```suggestion
        """ Returns log_2gs if it already exists in the dataframe, otherwise computes it using gamma_air """
        df = self.df0
        # TODO: deal with the case of gamma_self [so we don't forget] 
```